frontend/degree-plan/components/Search/SearchPanel.tsx

```diff
@@ -95,11 +100,17 @@ export const SearchPanel = ({ activeDegreeplanId }: { activeDegreeplanId: Degree
         setQueryString("");
     }, [ruleId])
 
+    const handleCloseSearch = () => {
+        setQueryString(""); 
+        setSearchPanelOpen(false);
+        setSearchedRuleId(-1);
```

Comment:
I think the value should be null instead of -1

Reply:
Yea makes sense